spark-tpcds-benchmark-runner/src/main/java/com/palantir/spark/tpcds/datagen/GenSortDataGenerator.java

```diff
@@ -41,6 +48,9 @@
     private static final Logger log = LoggerFactory.getLogger(GenSortDataGenerator.class);
 
     private static final int BYTES_PER_RECORD = 100;
+    private static final long BYTES_PER_GB = 1024 * 1024 * 1024;
```

Comment:
fine to do like this, but a handy thing to be aware of for this sort of thing: https://github.com/palantir/human-readable-types/blob/develop/human-readable-types/src/main/java/com/palantir/humanreadabletypes/HumanReadableByteCount.java